### ISSUE TITLE
When an empty hypha has a colon ":" in its name, it shows like there is some content inside 

### ISSUE THREAD

**Issue by danilax86** [bouncepaw/mycorrhiza#188]
![image](https://user-images.githubusercontent.com/46713109/214166429-66f0383c-5a2f-4b5c-b533-fd7a2ef44d7c.png)
![image](https://user-images.githubusercontent.com/46713109/214166456-ebec0569-745c-45eb-ba9c-295a7b8af959.png)

**Comment by danilax86** [comment]
Colon also is not rendered inside hypha as you can see

**Comment by bouncepaw** [comment]
Oh, it's actually a much deeper bug! When the parser finds a colon in the target of a link, it thinks it's a URL. The protocol is `Busy Doing Nothing`. Then it works for a couple of reasons.

As a workaround, I can recommend you to call this hypha //Busy Doing Nothing//, that's what they call this book anyway.

I will work on the bug.

**Comment by danilax86** [comment]
@bouncepaw so is this a mycomarkup thing?

**Comment by bouncepaw** [comment]
Yes. I'll move the issue to [Mycomarkup's issue tracker](https://todo.sr.ht/~bouncepaw/mycomarkup) later. I'm currently busy exploring your garden.

**Comment by danilax86** [comment]
@bouncepaw done

**Comment by bouncepaw** [comment]
Moved to https://todo.sr.ht/~bouncepaw/mycomarkup/12

Further discussion will be there.
